verilog/preprocessor/verilog_preprocess_test.cc

```diff
@@ -892,7 +895,18 @@ TEST(VerilogPreprocessTest, IncludingFileWithAbsolutePath) {
   const std::string equivalent_content =
       "module included_file(); endmodule\nmodule src(); endmodule\n";
 
-  VerilogPreprocess tester(VerilogPreprocess::Config({.include_files = true}));
+  // TODO(karimtera): allow including files with absolute paths.
+  // This is a hacky solution for now.
+  verilog::VerilogProject project(".", {"/"});
+  FileOpener file_opener =
```

Comment:
The nice thing with the abstraction of the file opening is, that you can also use that to simplify unit tests:
you could have a very simple implementation of the FileOpener without need for a VerilogProject; something along the line of

```c++
file_opener = [](absl::string_view filename) -> absl::StatusOr<absl::string_view> {
   if (filename == "foo") return "content of foo file";
   if (filename == "bar") return "content of bar file";
   return NotFoundError("file does not exist");
}
```

That way you can more fine-grained test things and don't even need to create a file in the filesystem.

Reply:
Yes, It's a good idea, I changed it in this test, but in other tests I wanted to keep the project to test the include directories with relative paths.